includes/query/SMW_Description.php

```diff
@@ -874,3 +874,111 @@ public function prune( &$maxsize, &$maxdepth, &$log ) {
 	}
 
 }
+
+/**
+ * Description of negation of descriptions.
+ *
+ * @ingroup SMWQuery
+ */
+class SMWNegation extends SMWDescription {
+	/**
+	 * @var SMWDescription
+	 */
+	protected $m_description;
```

Comment:
Real monies correlated in users whos ownership is undeniable  (ARIAD~ENX^^^^^ QUBBED>👊👍🌐♡♡♡♡♡.~ge6yhb6%×_+£88□□frTy76gi=%%%